services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -24,8 +24,20 @@
 router = APIRouter()
 logger = logging.getLogger(__name__)
 
+ServicesSelection = Set[Tuple[str, str]]
 
-@router.get("", response_model=List[ServiceOut])
+# These are equivalent to pydantic export models but for responses
+# SEE https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict
+# SEE https://fastapi.tiangolo.com/tutorial/response-model/#use-the-response_model_exclude_unset-parameter
+RESPONSE_MODEL_POLICY = {
+    "response_model_by_alias": True,
+    "response_model_exclude_unset": False,
+    "response_model_exclude_defaults": False,
+    "response_model_exclude_none": True,
```

Comment:
I would not have the exclude_none set. we will lose information that way

Reply:
done